paper.md

```diff
@@ -113,26 +113,25 @@ As mentioned above, the balance between pedagogy and working depends both on the
+> The *impostor phenomenon*, or *impostor syndrome* (IS) is a dissonant feeling experienced by certain high-achieving individuals, that despite objective evidence to the contrary, they are in fact not as intelligent or capable as they appear. Individuals with IS thus experience a fear of being "found out", shamed and expelled from their environment [@Clance1978-ef].
+> Initial observations of the first Astro Hack Week conducted by data science ethnographer Brittany Fiore-Gartland \footnote{http://astrohackweek.org/blog/ethnographic-notes.html} suggested that hack weeks are an environment prone to a particular kind of IS: participants might feel the need to be experts in multiple aspects of the activities pursued during a hack week: expertise in a scientific domain, as well as expertise in a the variety of technical tools used.  This particular form of IS hinges to some degree on the design focus on diversity of backgrounds (everyone else seems to know something that you do not!) and might be further exacerbated by the expectation that attendants expose their ideas to public scrutiny, find collaborators in a very short amount of time, and not least produce and present a successful hack at the end of the week (see also [this insightful blog post](https://medium.com/astronomy-without-stars/the-horror-of-hack-days-52c6b52cfc3b)).
+> The prevalence of IS at a hack week may be endemic to the format, and should thus be a major concern for any organizing committee. This is because of the chilling effect it tends to have on participants and the community as a whole, and particularly on women (in particular in fields in which women are under-represented) and members of ethnic and racial minorities, correlating with anxiety and other forms of mental distress [@Parkman2016-ro]. Less severely, another major concern is that IS inhibits risk taking: participants experiencing it will be less likely to ask a question, to put forward an idea for a hack, to be pro-active about forming new collaborations. Many of the goals of a hack week, including the successful completion of projects, lateral knowledge transfer, as well as community building are hampered.
+> We are working within our hack weeks to mitigate IS using various techniques. With respect to minority participants, ensuring adequate representation can decrease feelings of otherness and may help reduce IS. More generally, being open the presence and prevalence of IS can help participants feel more at ease (e.g. one Astro Hack Week participant remarked in a survey, _"I really appreciated the direct acknowledgement of impostor syndrome on the first day.  I think it helped ease the feeling!!"_) . Additionally, role models are very effective at encouraging positive behaviour: asking participants with prior experience at hack weeks at all academic levels to ask questions, even when they might know the answer, can help foster an inclusive environment that rewards risk taking.
```

Comment:
As far as I could tell, there are no studies that have addressed specifically the issue of prevention or mitigation of impostor syndrome. Being open about the presence and prevalence is in line with some of the research about stereotype threat (a much larger body of literature), but that research is a bit murky. I don't think role playing would hurt anyone, but I can't say there are any studies (that I could find) that tell us that it would help.